### DISCUSSION TITLE
SU2 compilation issues on Windows system.

### DISCUSSION THREAD

**Discussion by kasTnT** [su2code/SU2#2098] [Q&A]
I modified the code in the root directory's meson.build file by commenting out the 'if' statement for 'subdir('SU2_PY/pySU2')'. Then, I compiled it on a Linux system using the './meson.py build' and './ninja -C build install' commands. In the 'build' folder, I obtained the 'pysu2.py' and '_pysu2.so' files, and I was able to import 'pysu2' successfully. However, when I followed the same process on Windows, I modified the 'makefile.am' in the 'pySU2' directory by changing '.so' to '.dll' and updated the include paths. But the resulting '_pysu2.dll' didn't work as expected, and importing 'pysu2' resulted in an error, saying it couldn't find the function. When I inspected the 'dll' file using the 'dumbpin' command, I noticed it only contained a 'PyInit__pysu2' function. On the other hand, when using the nm command to inspect the _pysu2.so file compiled on Linux, I could see many functions. What's going on, and how can I correctly compile the 'SU2_CFD.py' script to run on Windows?

**Comment by kasTnT** [comment]
anybody？

**Reply by pcarruscag** [reply]
Few people are using windows for development, you will probably need to look for help from people who know the quirks of swig on windows

**Reply by kasTnT** [reply]
ok,thank u !
